### PR TITLE
Multi-tier priorities

### PR DESCRIPTION
**Pull request by avi-kik**
Allow defining arbitrary priorities for whitelist entries. When allocating available tx set capacity to whitelisted txs, higher priorities are allocated a proportionally-greater number of slots.

### CODE REVIEW

File: src/main/Whitelist.cpp [hunk 1]
```diff
@@ -16,92 +70,120 @@ Whitelist::getAccount()
...
+        auto data = df->getData();
+        auto name = data.dataName;
+        auto value = data.dataValue;
 
-	 // Iterate DataFrames to build the whitelist.
-	 // Structure: hash of (hint, vector of public key)
-	 // The hint is the last 4 bytes of the public key, and is used to
-	 // efficiently filter the possible entries for a given signature.
-     for (auto& df : dfs)
+        // If the value isn't 4 or 8 bytes long, skip the entry.
```
Comment (Ronserruya) [inline src/main/Whitelist.cpp]: What can an 8 bytes entry be?

**Comment by avi-kik** [comment]
Keep reading.

File: src/main/Whitelist.cpp (hunk 1 above)
Comment (avi-kik) [reply src/main/Whitelist.cpp]: Priority.

**Comment by oryband** [comment]
@avi-kik default branch was renamed to `master` from `kinecosystem/master`. Please update the base branch for this PR

**Pull request closed without merging**